### DISCUSSION TITLE
How do I include very large files?

### DISCUSSION THREAD

**Discussion by yodarunamok** [TheLastGimbus/GooglePhotosTakeoutHelper#340] [I NEED HELP] [answered]
There are a handful of videos in the output that are very large (a few GB), to the extent that Takeout exported them as entirely separate files. Where do I put these so that they get worked in with the rest? When extracted, all of the other files are dropped into directories that correspond to the corresponding year, but I don't know where these videos fit in the whole...

**Comment by yodarunamok** [comment, marked as answer]
I've done some more digging on this, and after locating one of videos, it appears that the name of the videos are mostly unchanged in the export -- the only change is an appended `-nnn` where the n's are replaced with  a number that indicates the export order. Also, I am unable to find any metadata for the videos in the Takeout JSON files, so I think this will have to be done manually. I'd be happy to be wrong about this last bit.

**Reply by TheLastGimbus** [reply]
unfortunetley, you may be right :(
